tifeatures/factory.py

```diff
@@ -576,20 +608,49 @@ async def items(
                 simplify=simplify,
             )
 
+            accept_header = request.headers.get("accept", "")
+
+            # CSV Response
+            if (f and f.value == "csv") or accept_header == "text/csv":
+                props = list(items[0].properties.keys()) + ["geometry"] if items else []
+                rows = [";".join(["collectionId", "itemId", *props]) + "\n"]
+                for f in items:
+                    rows.append(
+                        ";".join(
+                            list(
+                                map(
+                                    str,
+                                    [
+                                        collection.id,
+                                        f.id,
+                                        *f.properties.values(),
+                                        f.geometry.wkt,
+                                    ],
+                                )
+                            )
+                        )
+                        + "\n"
+                    )
+                return StreamingResponse(iter(rows), media_type="text/csv")
```

Comment:
to be reviewed
- we create the property list from the first items (we could in theory use the `Table.properties` method)

Reply:
I don't think we want the links for CSV, I think thta what you've got is enough